1-org/envs/shared/log_sinks.tf

```diff
@@ -34,88 +35,65 @@ resource "random_string" "suffix" {
   special = false
 }
 
-/******************************************
+module "logs_export" {
+  source = "../../modules/centralized-logging"
+
+  resources                      = local.parent_resources
+  resource_type                  = local.parent_resource_type
+  logging_destination_project_id = module.org_audit_logs.project_id
+
+  /******************************************
   Send logs to BigQuery
 *****************************************/
-
-module "log_export_to_biqquery" {
-  source                 = "terraform-google-modules/log-export/google"
-  version                = "~> 7.3.0"
-  destination_uri        = module.bigquery_destination.destination_uri
-  filter                 = local.main_logs_filter
-  log_sink_name          = "sk-c-logging-bq"
-  parent_resource_id     = local.parent_resource_id
-  parent_resource_type   = local.parent_resource_type
-  include_children       = true
-  unique_writer_identity = true
   bigquery_options = {
-    use_partitioned_tables = true
+    logging_sink_name          = "sk-c-logging-bq"
+    logging_sink_filter        = local.main_logs_filter
+    dataset_name               = "audit_logs"
+    partitioned_tables         = "true"
+    include_children           = true
+    expiration_days            = var.audit_logs_table_expiration_days
+    delete_contents_on_destroy = var.audit_logs_table_delete_contents_on_destroy
   }
-}
 
-module "bigquery_destination" {
-  source                     = "terraform-google-modules/log-export/google//modules/bigquery"
-  version                    = "~> 7.3.0"
-  project_id                 = module.org_audit_logs.project_id
-  dataset_name               = "audit_logs"
-  log_sink_writer_identity   = module.log_export_to_biqquery.writer_identity
-  expiration_days            = var.audit_logs_table_expiration_days
-  delete_contents_on_destroy = var.audit_logs_table_delete_contents_on_destroy
-}
-
-/******************************************
+  /******************************************
   Send logs to Storage
 *****************************************/
+  storage_options = {
+    logging_sink_filter          = local.all_logs_filter
+    logging_sink_name            = "sk-c-logging-bkt"
+    include_children             = true
+    storage_bucket_name          = "bkt-${module.org_audit_logs.project_id}-org-logs-${random_string.suffix.result}"
+    location                     = var.log_export_storage_location
+    retention_policy_is_locked   = var.log_export_storage_retention_policy == null ? null : var.log_export_storage_retention_policy.is_locked
+    retention_policy_period_days = var.log_export_storage_retention_policy == null ? null : var.log_export_storage_retention_policy.retention_period_days
+    force_destroy                = var.log_export_storage_force_destroy
+    versioning                   = var.log_export_storage_versioning
```

Comment:
As a follow up lets re evaluate the external variable surface for log-sinks to see if these should be exposed ( some like force_destroy maybe needed for test purposes).

Reply:
Sounds good!